autoscheduler/frontend/src/timeUtil.ts

```diff
@@ -0,0 +1,14 @@
+export const FIRST_HOUR = 8;
+export const LAST_HOUR = 21;
+/**
+ * Given the hours and minutes of a time, returns a formatted string representation of
+ * that time. By default, this uses 24-hour format, but an extra argument can be specified
+ * to use 12-hour format
+ * @param h hours
+ * @param m minutes
+ * @param use12Hour if true, applies 12-hour format instead of the default 24-hour format
+ */
+export function formatTime(h: number, m: number, use12Hour = false): string {
```

Comment:
So there's actually quite a few places that are using 24-hr time rather than 12-hr time. Instead of changing them in place, I'd say just change the default here to be to use12Hour time, and remove any place it's listed as true (see [this](https://github.com/aggie-coding-club/Rev-Registration/pull/160/commits/a8afbc8f25cbc5e70e228d55ac951f55df3b7f07#) commit)

Instead of listing the code places, you can see it showing 24-hr time here:

![chrome_3o3lGv7l64](https://user-images.githubusercontent.com/7295783/77189856-3dabb500-6aa6-11ea-85fa-d23bf1dd3fee.png)

It also uses 24-hr time for the MeetingCards.

```suggestion
export function formatTime(h: number, m: number, use12Hour = true): string {
```

Reply:
Oof, forgot about that